src/UserManagerSettings.ts

```diff
@@ -10,6 +10,8 @@ export const DefaultPopupWindowFeatures: PopupWindowFeatures = {
     location: false,
     toolbar: false,
     height: 640,
+    closeAutomaticallyPopupWindow: false,
```

Comment:
Please remove `popupWindowFeatures.closeAutomaticallyPopupWindow` and document that when `popupWindowFeatures.closePopupWindowAfter` > 0 its active, otherwise disabled